DIRECTORY.md

```diff
@@ -0,0 +1,159 @@
+# List of all files
+
+## DataStructures
+  * AATree
+    * [Aatree](https://github.com/TheAlgorithms/C-Sharp/blob/master/DataStructures/AATree/AATree.cs)
+    * [Aatreenode](https://github.com/TheAlgorithms/C-Sharp/blob/master/DataStructures/AATree/AATreeNode.cs)
```

Comment:
Not all classes represent stand-alone algorithms or data structures. For example, this one is used inside of AA Tree and shouldn't be displayed in this list.